h/blocklist/views.py

```diff
@@ -0,0 +1,36 @@
+from pyramid import httpexceptions
+from pyramid.view import view_config
+
+from h import models
+from h.api import search as search_lib
+
+
+@view_config(route_name='uriinfo',
+             request_method='GET',
+             accept='application/json',
+             renderer='json',
+             )
```

Comment:
Tiny thing, but putting this paren on the preceding line would be more consistent with the rest of the codebase.